### PR TITLE
Revisions SDL-0264 Separating the change of Audible status and the change of HMI Status

### PR DESCRIPTION
**Pull request by Shohei-Kawano**
## Introduction

Currently, there is no way to independently notify the Audible status to applications that output audio such as SDL Media.  
By changing the behavior of the existing API, the Audible state can be switched while maintaining the screen.

### CODE REVIEW

**Comment by Shohei-Kawano** [comment]
Hi @theresalech -san,
This time should be okay!
Could you check this pullrequest?

File: proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Separating the change of Audible status and the change of HMI Status
+
+* Proposal: [SDL-NNNN](NNNN-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: Awating Review
```
Comment (theresalech) [inline proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md]:
```suggestion
* Status: **Awaiting review**
```

File: proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+|state|result|
+|----|----|
+|Current|Change the HMI state of all applications currently either in FULL or LIMITED to (BACKGROUND, NOT_AUDIBLE)|
+|Changed|Keep the HMI state of all applications but change audible state of all applications to NOT_AUDIBLE|
+
+If HU wants to switch the screen (HMI Status) during PHONE_CALL, they can use API `BC.OnAppDeactivated (AppID)` and `BC.OnAppActivated (AppID)`.  
+
+## Potential downsides
+
+It is considered that there is an impact by continuing to display the SDL App screen when calling `OnEventChanged (PHONE_CALL, active: true)`.
```
Comment (theresalech) [inline proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status.md]:
```suggestion
It is considered that there is an impact by continuing to display the SDL app screen when calling `OnEventChanged (PHONE_CALL, active: true)`.  
```

**Comment by Shohei-Kawano** [comment]
Hi @theresalech -san
Thank you for your review.
I committed your suggestion.

**Pull request merged**